README.md

```diff
@@ -204,19 +206,29 @@ already implemented one that matches your need.
 
 
 5) **MetricHandler**:
-    This class is responsible for postprocess operations applied to prediction outputs.
-    This is performed for transforming the prediction outputs to a suitable format to be 
-    fed in metrics for computation. For example, while using BLEU in a language 
-    generation task, the prediction outputs need to be converted to string or 
-    list of strings, which is the behavior of the default implementation. However, for 
-    extractive question answering task in which the prediction outputs are returned as 
-    start and end indices pointing the answer within the context, additional information 
-    (e.g context in such case) may be needed, so before directly decoding the outputs in 
-    this case does not work, and additional operation needs to be done by converting 
-    prediction outputs to actual answer extracted from the context, you are able to do 
+    This class is mainly responsible for preprocessing applied to predictions and 
+    labels (references). This is performed for transforming the predictions and labels to a 
+    suitable format to be fed in metrics for computation. For example, while using BLEU in a 
+    language generation task, the predictions and labels need to be converted to a string or 
+    list of strings. However, for extractive question answering task in which the prediction 
+    outputs are returned as start and end indices pointing the answer within the context, 
+    additional information (e.g context in such case) may be needed, so before directly 
```

Comment:
This sentence should be fixed, at least starting from `so before directly `

Reply:
I rewrote that part.